### PR TITLE
clarify and simplify trip extraction rules

### PR DESCRIPTION
**Pull request by gabrielhora**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,68 +1,57 @@
...
 ## The trip extraction
 
-The central piece of the logbook is the trip extraction. Trip extraction is the process of 
-determining when a vehicle starts and stops moving based on data collected by gps trackers.
-
-
-When driving from A to B, a user will probably have interruptions like stopping at red lights, 
-traffic jams, tunnels. The expected behavior is to have one trip accounting for the whole 
-journey, and not broken trips like from A to first red light etc.
+The central piece of the logbook is the trip extraction. Trip extraction is the process of determining when a vehicle starts and stops moving based on data collected by gps trackers (Waypoints).
```
Comment (jeffhiltz) [inline readme.md]:
```suggestion
The central piece of the logbook is trip extraction. Trip extraction is the process of determining when a vehicle starts and stops moving based on data collected by GPS trackers. The data from the GPS is a collection of waypoints collected at semi-regular intervals.
```

File: readme.md [hunk 2]
```diff
@@ -1,68 +1,57 @@
...
 
+When driving from A to B, a user will probably have interruptions like stopping at red lights, traffic jams, tunnels. The expected behavior is to have one trip accounting for the whole journey, and not broken trips like from A to first red light etc.
```
Comment (jeffhiltz) [inline readme.md]:
```suggestion
When driving from A to B, a user will probably have interruptions to their movement like stopping at red lights and traffic jams. The expected behavior is to have one trip accounting for the whole journey, and not broken trips like from A to first red light etc.
```

File: readme.md [hunk 3]
```diff
@@ -1,68 +1,57 @@
...
 
-GPS technology has some limitations which the trip extraction should account for. 
-Within cities the GPS precision may be poorer than 15 meters and the GPS position might even "jump", 
-due to bad reception caused by reflections on building. There are places without gps signal
-like tunnels and underground parking lots.
-
+GPS technology has some limitations which the trip extraction should account for. Within cities the GPS precision may be poorer than 15 meters and the GPS position might even "jump", due to bad reception caused by reflections on building. There are places without gps signal like tunnels and underground parking lots.
```
Comment (jeffhiltz) [inline readme.md]:
```suggestion
GPS technology has some limitations which the trip extraction should account for. Within cities the GPS precision may be poorer than 15 meters and the GPS position might even "jump", due to bad reception caused by reflections on building. There are places without GPS signal like tunnels and underground parking lots.
```

**Review by jeffhiltz — COMMENTED** [review]
A few suggestions on the wording.

**Review by jeffhiltz — APPROVED** [review]
:goat: :house:

**Review by IoanaTeoP — APPROVED** [review]
Looks very good to me. The addition of the PNG is great, I think it will help the candidates understand the use case.

**Pull request merged**